### PR TITLE
[Merged by Bors] - chore: remove more unused variables

### PR DESCRIPTION
**Pull request by adomani**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Algebra/Module/LinearMap/Basic.lean [hunk 1]
```diff
@@ -21,7 +21,7 @@ open Function
 
 universe u u' v w x y z
 
-variable {R R₁ R₂ R₃ k S S₃ T M M₁ M₂ M₃ N₁ N₂ N₃ ι : Type*}
+variable {R R₂ S M M₂ : Type*}
```
Comment (grunweg) [inline Mathlib/Algebra/Module/LinearMap/Basic.lean:24]: Can you rename R2 to e.g. R' or R1? Right now, this looks a bit funky...

File: Mathlib/Algebra/Ring/Defs.lean [hunk 2]
```diff
@@ -47,7 +47,7 @@ assert_not_exists mul_rotate
 
 universe u v w x
 
-variable {α : Type u} {β : Type v} {γ : Type w} {R : Type x}
+variable {α : Type u} {R : Type x}
```
Comment (grunweg) [inline Mathlib/Algebra/Ring/Defs.lean:50]: Are v and w now unused universes? Can you rename the universe containing R to v then?

**Review by grunweg — COMMENTED** [review]
Thanks, just two minor comments.

File: Mathlib/Algebra/Ring/Defs.lean (hunk 2 above)
Comment (adomani) [reply Mathlib/Algebra/Ring/Defs.lean]: Universes renamed!

File: Mathlib/Algebra/Module/LinearMap/Basic.lean (hunk 1 above)
Comment (adomani) [reply Mathlib/Algebra/Module/LinearMap/Basic.lean]: I used `R R' M M'`.

**Comment by adomani** [comment]
Michael, thank you very much for the review!  I had added locally, but not pushed, some more removals, so the PR grew, but it is more of the same.  I will not add more modifications to this PR, though.

File: Mathlib/Algebra/Ring/Commute.lean [hunk 3]
```diff
@@ -23,7 +23,7 @@ For the definitions of semirings and rings see `Mathlib.Algebra.Ring.Defs`.
 
 universe u v w x
 
-variable {α : Type u} {β : Type v} {γ : Type w} {R : Type x}
+variable {R : Type x}
```
Comment (grunweg) [inline Mathlib/Algebra/Ring/Commute.lean:26]: Another chance to rename the universe :-)

File: Mathlib/Algebra/Group/Action/Opposite.lean [hunk 4]
```diff
@@ -97,7 +97,7 @@ In lemma names this is still called `op_vadd`. -/
 scoped notation3:73 m:73 " <+ᵥ " r:74 => AddOpposite.op r +ᵥ m
 
 section examples
-variable {α β : Type*} [SMul α β] [SMul αᵐᵒᵖ β] [VAdd α β] [VAdd αᵃᵒᵖ β] {a a₁ a₂ a₃ a₄ : α} {b : β}
+variable {β : Type*} [SMul α β] [SMul αᵐᵒᵖ β] [VAdd α β] [VAdd αᵃᵒᵖ β] {a a₁ a₂ a₃ a₄ : α} {b : β}
```
Comment (grunweg) [inline Mathlib/Algebra/Group/Action/Opposite.lean:100]: So alpha is already defined upstairs? You could also move \beta (just the type declaration) up...

**Review by grunweg — COMMENTED** [review]
Thanks! Two more comments on the new commits.

File: Mathlib/Algebra/Ring/Commute.lean (hunk 3 above)
Comment (adomani) [reply Mathlib/Algebra/Ring/Commute.lean]: I feel so omnipotent!

File: Mathlib/Algebra/Group/Action/Opposite.lean (hunk 4 above)
Comment (adomani) [reply Mathlib/Algebra/Group/Action/Opposite.lean]: Moved \beta next to \alpha

**Comment by grunweg** [comment]
Thanks!
maintainer merge

**Pull request closed without merging**